index.css

```diff
@@ -462,29 +535,392 @@ label {
+    font-size: 48px;
+    color: #172b4d;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    font-weight: 700;
+    width: 420px;
+    padding-bottom: 10px;
+  }
+
+  .section p {
+    color: #4d5c76;
+    font-size: 16px;
+    font-size: 1.75vw;
+    line-height: 24px;
+    letter-spacing: 0;
+    width: 620px;
+    font-weight: 500;
+    align-items: left;
+  }
+
+  .box1 {
+    background-image: url("images/Header bg.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 115%;
+    margin: auto;
+    height: 440px;
+    display: block;
+    justify-content: center;
+  }
+
+  .social-media ul li {
+    text-decoration: none;
+  }
+
   .article1,
   .article2,
   .article3,
   .article4 {
+    display: flex;
+    flex-direction: row;
+    width: 1156px;
+    height: 496px;
+    justify-content: space-between;
+    gap: 25px;
+    padding: 16px;
+    border: 1px solid #e3e5e9;
+    border-radius: 16px;
+    padding-bottom: 50px;
+  }
+
+  .show2 {
+    display: block;
+    padding: 50px 0 20px 0;
+  }
+
+  .article1 .fig1,
+  .article2 .fig1,
+  .article3 .fig1,
+  .article4 .fig1 {
+    display: none;
+  }
+
+  .article1 .fig2,
+  .article2 .fig2,
+  .article3 .fig2,
+  .article4 .fig2 {
+    display: block;
+    display: flexbox;
+    width: 544px;
+    height: 448px;
+    padding: 24px 24px 0 0;
+    object-fit: contain;
+    border-radius: 8px;
+  }
+
+  .float_box1 {
+    margin: 7px 1px 14px 0;
+  }
+
+  .article1 ul,
+  .article2 ul,
+  .article3 ul,
+  .article4 ul {
+    padding: 0;
+    display: flex;
+    gap: 10px;
+    list-style: none;
+  }
+
+  .about_section_align1 img {
+    padding: 10px 0 0 0;
+    background-color: #f7f7f9;
+    border: 1px solid none;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .about_self_section p {
+    color: #344563;
+    font-size: 16px;
+    font-size: 1.44vw;
+    line-height: 24px;
+    font-weight: 400;
+    text-align: justify;
+    justify-content: center;
+  }
+
+  .show1 {
+    display: none;
+  }
+
+  .ul1 {
+    margin: 15px 0 10px 0;
+  }
+
+  .article1div p,
+  .article2div p,
+  .article3div p,
+  .article4div p {
+    width: 100%;
+    height: 50px;
+    text-align: left;
+    color: #344563;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    justify-content: center;
+    gap: 25px;
+    margin-top: 10px;
+    padding-bottom: 120px;
+    padding-top: 40px;
+  }
+
+  .font2 {
+    width: 75px;
+    height: 24px;
+    color: #344563;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    padding: 1px 0;
+  }
+
+  .font3 {
+    width: 120px;
+    color: #6b778c;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    padding-left: 15px;
+  }
+
+  .font4 {
+    width: 295px;
+    color: #6070ff;
+    font-size: 12px;
+    align-items: left;
+    font-weight: 500;
+    row-gap: 25px;
+    padding: 16px 2px 2px 5px;
+  }
```

Comment:
- [ ] Kindly fix your font size on desktop version to do not overlap the others elements 

![image](https://user-images.githubusercontent.com/93630700/213876381-d756f276-cbe5-46a3-951c-b70aaebb46fb.png)

Reply:
Fixed the styling for aboutus form.